libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -16,6 +17,12 @@ AP_Camera_Backend::AP_Camera_Backend(AP_Camera &frontend, AP_Camera_Params &para
 // update - should be called at 50hz
 void AP_Camera_Backend::update()
 {
+    if (_params.options == 1 && AP::dal().get_armed()) {
```

Comment:
instead of comparing _param.options == 1 we should compare it to an enum value.. even if it just have a single entry in it.